content/docs/practices/instrumentation.md

```diff
@@ -206,11 +206,55 @@ Gauges can be set, go up, and go down. They are useful for snapshots of state,
 such as in-progress requests, free/total memory, or temperature. You should
 never take a `rate()` of a gauge.
 
-Summaries are similar to having two counters. They track the number of events
-*and* the amount of something for each event, allowing you to calculate the
-average amount per event (useful for latency, for example). In addition,
-summaries can also export quantiles of the amounts, but note that [quantiles are not
-aggregatable](http://latencytipoftheday.blogspot.de/2014/06/latencytipoftheday-you-cant-average.html).
+### Summary vs. histogram
+
+Summaries and histograms are more complex metric types. They both sample
+observations. They track the number of observations *and* the sum of the
+observed values, allowing you to calculate the average observed value (useful
+for latency, for example). Note that the number of observations (showing up in
+Prometheus as a time series with a `_count` suffix) is inherently a counter (as
+described above, it only goes up), while the sum of observations (showing up as
+a time series with a `_sum` suffix) is inherently a gauge (if a negative value
+is observed, it goes down).
+
+The essential difference is that summaries calculate streaming φ-quantiles on
+the client side and expose them, while histograms count observations in buckets
+and expose those counts. Calculation of quantiles from histograms happens on the
+server side using the [`histogram_quantile()`
+function](/docs/querying/functions/#histogram_quantile()).
+
+Both approaches have specific advantages and disadvantages:
+
+|   | Histogram | Summary
+|---|-----------|---------
+| Configuration | Need to configure buckets suitable for the expected range of observed values. | Need to configure φ-quantiles and sliding window, other φ-quantiles and sliding windows cannot be calculated later.
+| Client performance | Observations are very cheap as they only need to increment counters. | Observations are expensive due to the streaming quantile calculation.
+| Server performance | Calculating quantiles is expensive, consider [recording rules](/docs/querying/rules/#recording-rules) as a remedy. | Very low resource needs.
+| Number of time series | Low for Apdex score (see below), very high for accurate quantiles. Each bucket creates a time series. | Low, one time series per configured quantile.
+| Accuracy | Depends on number and layout of buckets. Higher accuracy requires more time series. | Configurable. Higher accuracy requires more client resources but is relatively cheap.
+| Specification of φ-quantile and sliding time window | Ad-hoc in Prometheus expressions. | Preconfigured by the client.
+| Aggregation | Ad-hoc aggregation with [Prometheus expressions](/docs/querying/functions/#histogram_quantile()). | In general [not aggregatable](http://latencytipoftheday.blogspot.de/2014/06/latencytipoftheday-you-cant-average.html).
+
```

Comment:
Sure. But I have to argue with 100+ Soundcloud engineers, some of them very opinionated. So I'll prefer not to open a discussion front here. Stating that you can't aggregate pre-calculated quantiles in a meaningful way is a much more convincing argument (because of... math!) then opening that can of worms in which situation you want which level of accuracy...